_test/test_utilities/test_ortho.m

```diff
@@ -0,0 +1,46 @@
+classdef test_ortho < TestCase
+    properties
+    end
+    methods
+        function obj = test_ortho(varargin)
+            if nargin<1
+                name = 'test_ortho';
+            else
+                name = varargin{1};
+            end
+            obj = obj@TestCase(name);
+        end
+        function test_ortho_throws(~)        
+            u = [0,0,0];
+            assertExceptionThrown(@()ortho_vec(u),'HERBERT:ortho_vec:invalid_argument')
+        end
+        
+        function test_ortho3(~)        
```

Comment:
for all 3 tests, worth having more descriptive names. So (I think) test1 is a basic check that 2 orthogonal vectors are produced? Are 2 & 3 just doing the same for different input directions? Is there anything special about the choice of u in each case. Someone in the future is going to look at these tests, and think "why these particular ones?" Naming of the tests and comments will help reassure the maintainter what is expected here.